### DISCUSSION TITLE
Honeypot-multiple NIC or networks

### DISCUSSION THREAD

**Discussion by mhgoff** [Security-Onion-Solutions/securityonion#7550] [Unsupported Versions] [answered]
Is it possible to have multiple NICs on the honeypot?  Say I have 5 vlans on my corporate network....Can I have a single device with 5 NICs or would I need to create 5 devices?
Or can I have a management interface plus a honeypot interface?

**Comment by defensivedepth** [comment, marked as answer]
This is not currently possible, but will consider it in the future.

**Reply by craigsmooth** [reply]
Thanks Josh. So, if I understand correctly. The Honeypot will only listen on the management inferface. Is that correct?

**Reply by defensivedepth** [reply]
Yes that is correct. Sounds like there is a desire to have a management interface and then a "honeypot" interface?

**Reply by craigsmooth** [reply]
Absolutely. The way we deploy our sensors is the mgmt interface is on an isolated mgmt vlan that is not "user facing" so we would get no real benefit from deploying the honeypot. If we had the ability to bind the honeypot service/container to a different physical nic and then patch that into a user vlan or server vlan, we could certainly benefit from this new feature. I would imagine many security onion users would also be in the same situation. Great feature and looking forward to see how it evolves and hopefully we will be able to use it in the future.

**Reply by mhgoff** [reply]
What craigsmooth stated is my exact situation as well.

**Reply by defensivedepth** [reply]
@craigsmooth @mhgoff I have created an issue to track this: https://github.com/Security-Onion-Solutions/securityonion/issues/7589
